frontend/pages/queries/edit/components/EditQueryForm/EditQueryForm.tsx

```diff
@@ -836,6 +897,23 @@ const EditQueryForm = ({
               >
                 Observers can run
               </Checkbox>
+              {isPremiumTier && (
+                <TargetLabelSelector
+                  selectedTargetType={selectedTargetType}
+                  selectedCustomTarget=""
```

Comment:
Maybe a little clearer to make this prop optional?

Reply:
Done!